var/spack/repos/builtin/packages/spfft/package.py

```diff
@@ -60,6 +60,12 @@ class Spfft(CMakePackage, CudaPackage):
     # FindHIP cmake script only works for < 4.1
     depends_on('hip@:4.0', when='@:1.0.1 +rocm')
 
+    # Propagate openmp to blas
```

Comment:
SpFFT does not depend on BLAS at all, so I think these changes should be included for SPLA instead.

Reply:
Yes, that was intended for spla indeed.